### PR TITLE
network devices claim

### PR DESCRIPTION
**Pull request by kikumar552**

### CODE REVIEW

File: internal/provider/networks_devices_claim_resource.go [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+	// Now set the updated state of the resource.
+	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
+
+	// Log that the resource was updated.
+	tflog.Trace(ctx, "updated resource")
+}
+
+// Delete function is responsible for deleting a resource.
+// It uses a DeleteRequest and responds with a DeleteResponse which contains the updated state of the resource or an error.
+func (r *NetworksDevicesClaimResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
```
Comment (iamdexterpark) [inline internal/provider/networks_devices_claim_resource.go:219]: So there is a API call to delete a single device: {{baseUrl}}/networks/:networkId/devices/remove

What we can do here is loop through data.serials and make that API call, then remove from state.

File: internal/provider/networks_devices_claim_resource_test.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+					resource.TestCheckResourceAttr("meraki_network.test", "product_types.0", "appliance"),
+					resource.TestCheckResourceAttr("meraki_network.test", "product_types.1", "switch"),
+					resource.TestCheckResourceAttr("meraki_network.test", "product_types.2", "wireless"),
+					resource.TestCheckResourceAttr("meraki_network.test", "notes", "Additional description of the network"),
+				),
+			},
+
+			// Claim and Read NetworksDevicesClaim
+			{
+				Config: testAccNetworksDevicesClaimResourceConfigCreate(os.Getenv("TF_ACC_MERAKI_MX_SERIAL")),
```
Comment (iamdexterpark) [inline internal/provider/networks_devices_claim_resource_test.go:51]: ACC tests look great, Only thing is to add the ENV VARS for the MX Serial to the repos secrets.

File: internal/provider/networks_devices_claim_resource.go (hunk 1 above)
Comment (iamdexterpark) [reply internal/provider/networks_devices_claim_resource.go:219]: You should also be able to delete a network which contains serials without issue but when it comes to the organization there isn't a straightforward way to unclaim it. We may have to use the organization claim resource to move it intio our static "DO_NOT_DELETE" org so we can delete the ephemeral test_acc organization made during the tests:

https://community.meraki.com/t5/Developers-APIs/Unclaim-devices-from-an-ORG-API/m-p/76010/highlight/true

Comment (kikumar552) [reply internal/provider/networks_devices_claim_resource.go:219]: Understand I added accordingly to move the stuff to DO_NOT_DELETE org id

**Review by iamdexterpark — APPROVED** [review]
LGTM

**Pull request merged**